editor/src/components/navigator/navigator-item/render-prop-selector-popup.tsx

```diff
@@ -77,6 +86,12 @@ interface RenderPropPickerProps {
   id: string
 }
 
+interface PreferredChildToInsert {
+  elementToInsert: (uid: string) => JSXElementChild
+  additionalImports: Imports | null
```

Comment:
do we really need the null? what is the difference between null and empty object?

Reply:
Turns out we don't need the `| null`, fixed it in https://github.com/concrete-utopia/utopia/pull/5050/commits/9f522b69ad78d99d72d84521c8f50d9e8f3f7f50